documentation/source/building.rst

```diff
@@ -292,6 +292,18 @@ The following variables are makefile variables, not environment variables.
       to set the random seed value if :envvar:`RANDOM_SEED` is not set.
       If both ``+ntb_random_seed`` and ``+seed`` are set, ``+ntb_random_seed`` is used.
 
+.. make:var:: CMD_PREFIX
+
+      Prefix added to invocations of $(CMD)
+
+      Can be used to add environment variables or prefixed commands to invocations of
+      $(CMD). This feature has been motivated by `issue 644 <https://github.com/cocotb/cocotb/issues/644>`_
+      to work work around issues with conflicting dynamic libraries. In these situations
+      users can set ``CMD_PREFIX := LD_PRELOAD="foo.so bar.so"`` to force a particular
+      library.
```

Comment:
Convention is 1 sentence per line and I don't think you should mention motivation in reference documentation.

```suggestion
      Can be used to add environment variables or prefixed commands to invocations of $(CMD).
      For example, ``CMD_PREFIX='LD_PRELOAD="foo.so bar.so"'`` can be used to force a particular library to load.
      Or, ``CMD_PREFIX="gdb --args"`` to run the simulation with the GDB debugger.
```